ee/tabby-webserver/src/routes/repositories.rs

```diff
@@ -0,0 +1,66 @@
+use anyhow::Result;
+use axum::{extract::Path, http::StatusCode, response::Response, Json};
+use tabby_common::{path::repositories_dir, SourceFile};
+use tracing::{debug, instrument, warn};
+
+use crate::repositories::{resolve_dir, resolve_file, Repository, DATASET};
+
+#[utoipa::path(
+    get,
+    path = "/repositories/{name}/resolve/{path}",
+    operation_id = "resolve",
+    responses(
+        (status = 200, description = "Success"),
+        (status = 404, description = "Not Found"),
+        (status = 500, description = "Internal Server Error"),
+    )
+)]
```

Comment:
There's no need to provide utoipa documentation since we're not exposing the api doc